### DISCUSSION TITLE
LaTeX Math: How to get nice italicized classic LaTeX style Computer Modern fonts?

### DISCUSSION THREAD

**Discussion by brendanjmeade** [quarto-dev/quarto-cli#9009] [Q&A] [answered]
### Description

I'm just started using Quarto and I think it's absolutely fantastic!  I'm looking for assistance on how to get math in slides to render in the nice italicized classic LaTeX style Computer Modern font style.  Currently, I'm getting what you see in the screenshot below.  I'm not sure what style it is, but it sort of looks like MS Word style rather than LaTeX style.  The following screenshot shows this:

<img width="400" alt="default_math_font" src="https://github.com/quarto-dev/quarto-cli/assets/4225359/5a899849-89e5-4ddf-8d68-3dcdad1e1fcb">

I'm hoping to get math that renders in the following style,

<img width="263" alt="latex_math_font" src="https://github.com/quarto-dev/quarto-cli/assets/4225359/d6b260f1-c541-4f30-abe8-b88043e1e53f">

The content of the `.qmd` that generated this first screenshot is pasted below.  I've tried changing `method: mathjax` to `method:katex` and get the same result.

I'm using Quarto 1.4.550 on MacOS 14.2.1.  Guidance and help much appreciated!

````qmd
---
title: "math example"
format:
  revealjs: 
    controls: true
    theme: simple
    smaller: true
    scrollable: true
    slide-number: true
    controls-layout: bottom-right
  html:
    html-math-method:
      method: mathjax
      url: "https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"
---

### A math example
Centered equation,
$$
\frac{dP}{dt} = \sin(\alpha P).
$$
Inline equation, $dP/dt = \sin(\alpha P)$.
````

**Comment by mcanouil** [comment, marked as answer]
This is MathJax configuration.
You can tweak CSS, manually change renderer.
<img width="1045" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/50238d9c-bdda-4386-a78f-3ea1b73acb5c">

For more, please take a look at MathJax documentation: https://docs.mathjax.org/en/latest/options/menu.html#the-configuration-block

The default renderer is not the same according to MathJax version.
This matters because Reveal.js embed an old version of MathJax.

You can simply set your MathJax configuration globally:

````qmd
---
title: "math example"
format:
  revealjs: default
  html: default
html-math-method:
  method: mathjax
  url: "https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"
---

## A math example

Centered equation,

$$
\frac{dP}{dt} = \sin(\alpha P).
$$

Inline equation, $dP/dt = \sin(\alpha P)$.
````

**Reply by brendanjmeade** [reply]
This completely solve the issue.  Thank you!
